ui_test/src/rustc_stderr.rs

```diff
@@ -87,6 +87,9 @@ impl RustcMessage {
                 messages.resize_with(line + 1, Vec::new);
             }
             messages[line].push(msg);
+        // All other messages go into the general bin, unless they are specifically of the
+        // "aborting due to X previous errors" variety, as we never want to match those. They
+        // only count the number of errors and provide no useful information about the tests.
```

Comment:
Oh I see.

Miri will also sometimes talk about the program having been "aborted", so I am a bit worried about matching the wrong diagnostic here... can you add `&& ends_with("previous errors")` or does that not work?

Reply:
I added something that will work for both the plural and singular version, but I can also make it 100% robust by checking both variants

Reply:
This looks good, thanks!